sweep/sweeper.go

```diff
@@ -282,6 +341,27 @@ func (s *UtxoSweeper) SweepInput(input input.Input) (chan Result, error) {
 	return sweeperInput.resultChan, nil
 }
 
+// validateFeePreference validates a client's fee preference in accordance to
+// the UtxoSweeper's config.
+func (s *UtxoSweeper) validateFeePreference(
+	feePreference FeePreference) (lnwallet.SatPerKWeight, error) {
+
+	feeRate, err := DetermineFeePerKw(s.cfg.FeeEstimator, feePreference)
```

Comment:
`DetermineFeePerKw` internally uses a hard coded `FeePerKwFloor`, while we also have a min relay fee in the sweeper. These are the same thing, aren't they? If so, shouldn't the same source be used?

Reply:
Nodes can have custom minimum relay fee rates, so it's possible that it's actually above the fee floor we enforce. In any case, we really care about the minimum relay fee rate in this case.